src/styles/index.scss

```diff
@@ -1,3 +1,104 @@
+$font-color-primary: #060b35;
+$font-color-collateral: #616070;
+$background-color:#fff;
+$border-color: #f3f3f3;
+$button-color:#00acdc;
+$hug: 16px;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
 body {
+  background-color: $background-color;
+  display: flex;
+  justify-content: center;
   margin: 0;
+  padding: 0;
+}
+
+.card {
```

Comment:
Create separate files for `card` and `stars`
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/c26d0fde-985d-407b-9850-35493bae80c7)

Reply:
done, thanks)